index.bs

```diff
@@ -796,9 +821,9 @@ objects also cannot implement arbitrary sets of interfaces.
 Objects can be defined to implement a single given interface |A|,
 which means that it also implements all of |A|’s
 [=inherited interfaces=].  In addition,
-an [=implements statement=] can be
+an [=includes statement=] can be
 used to define that objects implementing an interface will always
-also implement another interface.
+also implement a [=mixin=]
```

Comment:
Maybe we should avoid the verb "implement" for mixins, in favor of e.g. "include the members defined by that mixin" or similar?

Reply:
Absolutely. I need to check I didn't forget any. Thanks for catching this one.

Reply:
Oops, didn't check context before answering this. This "implement" is related to JS objects implementing an interface, not interfaces implementing mixins as I initially thought. This is worth pondering a bit to find the right terminology here.

Reply:
Modified the sentence slightly. But still kept the "implements" terminology, which really describes platform objects implementing interfaces and their members, not interfaces implementing other [NIO] interfaces.